bpfman/src/bpf.rs

```diff
@@ -616,22 +616,21 @@ impl BpfManager {
                             "bpfman-ns"
                         };
```

Comment:
Andrew:
I really don't think we should be bringing k8s specific things into core bpfman... Maybe we make this path configurable?

Andre:
I agree in general, but it's more user friendly if you don't have to configure it.

Reply:
I mean yes but for a true separation of concerns bpfman as the core process shouldn't be aware of K8s unless it really needs to be, that's the whole point of the agent IMO

Anyways It's not a hill I'm willing to die on at the moment.